hello_phoenix/firmware/lib/firmware/application.ex

```diff
@@ -3,29 +3,45 @@ defmodule Firmware.Application do
   # for more information on OTP Applications
   @moduledoc false
 
-  @target Mix.Project.config()[:target]
-
   use Application
 
   def start(_type, _args) do
     # See https://hexdocs.pm/elixir/Supervisor.html
     # for other strategies and supported options
     opts = [strategy: :one_for_one, name: Firmware.Supervisor]
-    Supervisor.start_link(children(@target), opts)
+
+    children =
+      [
+        # Children for all targets
+        # Starts a worker by calling: Firmware.Worker.start_link(arg)
+        # {Firmware.Worker, arg},
+      ] ++ children(target())
+
+    Supervisor.start_link(children, opts)
   end
 
   # List all child processes to be supervised
-  def children("host") do
+  def children(:host) do
     [
+      # Children that only run on the host
       # Starts a worker by calling: Firmware.Worker.start_link(arg)
       # {Firmware.Worker, arg},
     ]
   end
 
   def children(_target) do
+    :os.cmd('epmd -daemon')
```

Comment:
gotta have the Erlang distribution 🤣

Reply:
ah whoops, i meant to remove that. I was testing that it worked in the live dashboard..